engine/src/main/java/org/kigalisim/engine/serializer/EngineResultBuilder.java

```diff
@@ -0,0 +1,262 @@
+   * @return This builder for method chaining
+   */
+  public EngineResultBuilder setImportConsumptionValue(EngineNumber importConsumptionValue) {
+    this.importConsumptionValue = importConsumptionValue;
+    return this;
+  }
+
+  /**
+   * Set the recycle consumption value.
+   *
+   * @param recycleConsumptionValue The recycle consumption value in tCO2e 
+   *     or equivalent
+   * @return This builder for method chaining
+   */
+  public EngineResultBuilder setRecycleConsumptionValue(EngineNumber recycleConsumptionValue) {
+    this.recycleConsumptionValue = recycleConsumptionValue;
+    return this;
+  }
+
+  /**
+   * Set the population value.
+   *
+   * @param populationValue The population value in terms of equipment
+   * @return This builder for method chaining
+   */
+  public EngineResultBuilder setPopulationValue(EngineNumber populationValue) {
+    this.populationValue = populationValue;
+    return this;
+  }
+
+  /**
+   * Set the population new value.
+   *
+   * @param populationNew The amount of new equipment added this year
+   * @return This builder for method chaining
+   */
+  public EngineResultBuilder setPopulationNew(EngineNumber populationNew) {
+    this.populationNew = populationNew;
+    return this;
+  }
+
+  /**
+   * Set the recharge emissions value.
+   *
+   * @param rechargeEmissions The greenhouse gas emissions from recharge activities
+   * @return This builder for method chaining
+   */
+  public EngineResultBuilder setRechargeEmissions(EngineNumber rechargeEmissions) {
+    this.rechargeEmissions = rechargeEmissions;
+    return this;
+  }
+
+  /**
+   * Set the end-of-life emissions value.
+   *
+   * @param eolEmissions The greenhouse gas emissions from end-of-life equipment
+   * @return This builder for method chaining
+   */
+  public EngineResultBuilder setEolEmissions(EngineNumber eolEmissions) {
+    this.eolEmissions = eolEmissions;
+    return this;
+  }
+
+  /**
+   * Set the energy consumption value.
+   *
+   * @param energyConsumption The energy consumption value
+   * @return This builder for method chaining
+   */
+  public EngineResultBuilder setEnergyConsumption(EngineNumber energyConsumption) {
+    this.energyConsumption = energyConsumption;
+    return this;
+  }
+
+  /**
+   * Set the import supplement data.
+   *
+   * @param importSupplement Supplemental import information
+   * @return This builder for method chaining
+   */
+  public EngineResultBuilder setImportSupplement(ImportSupplement importSupplement) {
+    this.importSupplement = importSupplement;
+    return this;
+  }
+
+  /**
+   * Check that the builder is complete and create a new result.
+   *
+   * @return The result built from the values provided to this builder
+   * @throws IllegalStateException if any required field is missing
+   */
+  public EngineResult build() {
+    checkReadyToConstruct();
+    return new EngineResult(
+      application, substance, year, manufactureValue, importValue, recycleValue,
```

Comment:
Please have one variable passed per line for this form of line continuation. So, please follow this pattern:

```
return new EngineResult(
    application,
    substance,
    // …
);
```